packages/cozy-harvest-lib/src/components/OAuthWindow.jsx

```diff
@@ -95,9 +97,10 @@ export class OAuthWindow extends PureComponent {
    * * realtime message from web apps (see handleMessage)
    * * url changes from mobile apps
    *
-   * @param  {String} data.key - `io.cozy.accounts` id The created OAuth account
-   * @param  {String} data.error - error message
-   * @param  {String} data.oAuthStateKey - key for localStorage
+   * @param  {Object} data
+   * @param  {String|null} data.key - `io.cozy.accounts` id The created OAuth account
+   * @param  {String|null} data.error - error message
+   * @param  {String|null} data.oAuthStateKey - key for localStorage
```

Comment:
Are you really sure that this `null` and not `undefined`? 

If this is `undefined` then, the doc should be written like this: 
```
 @param  {String} [data.key] - `io.cozy.accounts` id The created OAuth account
```